### PR TITLE
Hero: Add "Vertically align slide contents" setting

### PR DESCRIPTION
**Pull request by AlexGStapleton**
Resolves https://github.com/siteorigin/so-widgets-bundle/issues/986

@Misplon You mentioned you wanted to see how it works before working on the description. How does the following (very rough suggestion) sound?

> Please note that Extra top padding may cause the slide contents to be slightly off-centered.

### CODE REVIEW

File: widgets/hero/styles/default.less [hunk 1]
```diff
@@ -55,6 +56,12 @@
 			max-width: @slide_width;
 			height: @slide_height;
 
+			& when (@vertically_align = true) {
+				display: flex;
+				justify-content: center;
+				flex-direction: column;
```
Comment (Misplon) [inline widgets/hero/styles/default.less]: What do you think about moving to alphabetical declarations? _s uses that style and I have been in the themes for a long time. `display, flex-direction and then justify-content` in this case. It's a simple organization method that's easy for everyone to follow.

**Comment by Misplon** [comment]
@AlexGStapleton How about:

_For perfect centering, consider setting the Extra top padding setting to 0 when using this setting._

**Comment by Misplon** [comment]
Otherwise, it looks all good mate.

**Comment by AlexGStapleton** [comment]
> 
> 
> @AlexGStapleton How about:
> 
> _For perfect centering, consider setting the Extra top padding setting to 0 when enabling this setting._

I think that's fine. Adding now.

File: widgets/hero/styles/default.less (hunk 1 above)
Comment (AlexGStapleton) [reply widgets/hero/styles/default.less]: Sounds good. I'll do this now.

**Pull request merged**